helper/pq/pq.go

```diff
@@ -18,6 +19,10 @@ func (Helper) GetTablesQuery() string {
 	return queryAllTables
 }
 
-func (Helper) TruncateTableCommand(tableName string) string {
-	return fmt.Sprintf("TRUNCATE TABLE %s", tableName)
+func (Helper) TruncateTablesCommand(tableNames []string) string {
+	if len(tableNames) == 0 {
```

Comment:
I prefer to test `tableNames` array from consumer of this method instead. Otherwise we can return an error like
```
var EmptyTablesListError = errors.New("List of tables must not be empty")
```